src/fs.rs

```diff
@@ -0,0 +1,19 @@
+use std;
+use std::fs::File;
+use std::io::Read;
+use std::path::Path;
+
+#[allow(dead_code)]
+pub fn read_file_sync(path: &Path) -> std::io::Result<String> {
+  File::open(path).and_then(|mut f| {
+    let mut contents = String::new();
+    f.read_to_string(&mut contents)?;
```

Comment:
This is not generally safe, because a file's contents are not necessarily valid utf-8.
We'll make it read to a buffer later? Otherwise consider renaming the function to `read_text_file_sync`.

Reply:
Oh - I thought Strings were like `std::string`. Sigh. Will change...

Reply:
This gonna stay as it is?